paper-dropdown-menu.html

```diff
@@ -242,6 +242,16 @@
           verticalAlign: {
             type: String,
             value: 'top'
+          },
+
+          /**
+           * If you're using PaperInputBehavior to implement your own paper-input-like
+           * element, bind this to the `<input is="iron-input">`'s `readonly` property.
+           */
+          readonly: {
+            type: Boolean,
+            value: false,
+            observer: '_readonlyChanged'
```

Comment:
For the actual weight, `_readonlyChanged` callback is called the first time when `readonly` is set to its default value, and each time it changes value, which means when is set to true, we have `_readonlyChanged` called twice.
If you remove the default value for the `readonly` property, `_readonlyChanged` will be called only once, when we it is set the first time.